src/components/pages/sign-up/sign-up-form.tsx

```diff
@@ -25,14 +29,20 @@ export default function SignUpForm() {
     register,
     formState: { errors },
     getValues,
-  } = useForm<FormValues>({ defaultValues });
+  } = useForm<SignUpFormValues>({ defaultValues });
 
-  function onSubmit(values: FormValues) {
-    console.log('submit', values);
+  function handleSignUp({ email, newPassword }: SignUpFormValues) {
+    userPool.signUp(email, newPassword, [], [], (error, data) => {
+      if (error) {
+        return console.error(error);
+      }
```

Comment:
P5: 다음과 같이 console.error와 return을 따로 작성하는 것에 대해선 어떻게 생각하시나요? userPool.signUp 함수의 return 타입이 void일 것 같은데, 위와 같이 console.error를 반환하도록 작성되어 있다면 'console 객체를 받아서 userPool.signUp이 추가 로직을 수행하나?'라고 생각할 수도 있을 것 같아서요. 단순 저의 취향이니 무난하게 넘어가셔도 좋을 것 같습니다!

```suggestion
      if (error) {
        console.error(error);
        return;
      }
```

Reply:
이해 됐습니다~~ 확실히 리턴문과 에러를 나누는 것이 더 잘 와닿는 것 같아요!